### ISSUE TITLE
Cannot train entities in Interactive Learning on new Interface.

### ISSUE THREAD

**Issue by akshayrana30** [RasaHQ/rasa#3581]
**Rasa version**:1.0

**Python version**: 3.6

**Operating system** (windows, osx, ...):Windows/Ubuntu

**Issue**: **Allow to modify or add detected entities in Interactive Learning in the new user interface.** A similar issue #1972 was raised last year for command line interactive learning and has been very helpful. The same should have been there in the new interface as well. The ability to add/modify entities in interactive learning.

**Content of configuration file (config.yml)**:

<pre>pipeline:

*   name: rasa_nlu.tokenizers.whitespace_tokenizer.WhitespaceTokenizer
*   name: rasa_nlu.extractors.crf_entity_extractor.CRFEntityExtractor
*   name: rasa_nlu.extractors.entity_synonyms.EntitySynonymMapper
*   name: rasa_nlu.featurizers.count_vectors_featurizer.CountVectorsFeaturizer
     token_pattern: (?u)\b\w+\b
*   name: rasa_nlu.classifiers.embedding_intent_classifier.EmbeddingIntentClassifier
*   name: rasa_nlu.extractors.duckling_http_extractor.DucklingHTTPExtractor
     url: [http://35.233.245.64](http://35.233.245.64)
     dimensions:
*   number
     language: en
     policies:
*   name: KerasPolicy
     epochs: 100
     max_history: 5
*   name: FallbackPolicy
     fallback_action_name: action_default_fallback
*   name: MemoizationPolicy
     max_history: 5
*   name: FormPolicy
     </pre>

**Content of domain file (domain.yml)** (if used & relevant):

**Comment by akelad** [comment]
@akshayrana1139 could you clarify what you mean? That's possible in interactive learning right  now, it'll ask you to annotate your entities before confirming

**Comment by akshayrana1139** [comment]
![Interactive Learning](https://user-images.githubusercontent.com/29054015/58323339-4e1ce000-7e41-11e9-9d1f-aad3289b4e32.JPG)

There is no way to add/modify entities. I also tried selecting the word to get a pop-up to annotate entity as we get in the NLU training.

**Comment by akelad** [comment]
ahhh sorry i misread your issue, yeah we need to fix that -- @gausie are we aware of this?

**Comment by tmbo** [comment]
This should already work on other screens and the possibility to label entities should be added to this screen as well.

**Comment by akelad** [comment]
by "should be added to this screen as well" we mean it'll be part of an upcoming release

**Comment by gausie** [comment]
A little late, but yes this is coming soon.

**Comment by wochinge** [comment]
@gausie That's sth we are still working on, right?

**Comment by mehtajinesh** [comment]
any updates on this issue? resolved yet?

**Comment by wochinge** [comment]
@mehtajinesh This is not yet done. @eibhleag @gausie Can you please give some feedback if / when we are tackling this?

**Comment by eibhleag** [comment]
This specific workflow isn't in our immediate backlog, but the Conversations view in Rasa X will soon allow you to filter for Interactive Learning conversations (currently they're hidden), so it'll be possible to annotate entities that way 🙂

**Comment by shalabhsingh** [comment]
any updates on this?

**Comment by alexweidauer** [comment]
Thank you for raising this issue about Rasa X. We decided to stop supporting the Community Edition (free version) of ‘Rasa X’ (see more info here: https://rasa.com/blog/rasa-x-community-edition-changes/). That’s why we’ve closed your issue. We suggest that Rasa Enterprise customers raise the issue with our customer support team if they haven’t done so already.
